eng/pipelines/eng-common-sync.yml

```diff
@@ -20,8 +20,9 @@ jobs:
     steps:
     - template: ./templates/steps/sync-directory.yml
       parameters:
-        CommitMessage: Sync eng/common directory with azure-sdk-tools repository
-        DirectoryToSync: eng/common
+        CommitMessage: "Sync eng/common directory with azure-sdk-tools repository for Tools PR #$(System.PullRequest.PullRequestNumber)"
+        DirectoryToSync: "eng/common"
+        PRBranchName: "$(System.PullRequest.SourceBranch)-$(System.PullRequest.PullRequestNumber)"
```

Comment:
Why is the SourceBranch interesting? We should definitely add something like "sync-eng-common" to the branch name so we know what the branch is for.

Reply:
Since we will be having different syncs PRs for different PRs in the tools repo, I think the SourceBranch will be helpful for identifying the matching PR on the tools repo.

Reply:
I was hoping the PR number would be enough but I don't mind having the SourceBranch as well.